sdks/java/core/src/main/java/org/apache/beam/sdk/options/GoogleApiDebugOptions.java

```diff
@@ -40,7 +40,7 @@
    * to Google, specifically to the Google Cloud Dataflow team.
    */
   @Description("This option enables tracing of API calls to Google services used within the "
-      + "Dataflow SDK. Values are expected in JSON format {\"ApiName\":\"TraceDestination\",...} "
+      + "Apache Beam SDK. Values are expected in JSON format {\"ApiName\":\"TraceDestination\",...} "
```

Comment:
moved Apache up a line so spaces stay confined to the end of the lines, still squeaks by at 100 chars